main.go

```diff
@@ -30,6 +30,7 @@ var (
 	flagBindAddress    string = "0.0.0.0:5000"
 	flagSmtpHostname   string = "127.0.0.1"
 	flagSmtpPort       int    = 25
+	flagHelloHostName  string = "localhost"
```

Comment:
Please add the `Smtp` as the prefix of the variable name to follow the current convention of smtp config. I would also suggest a shorter name. E.g. `flagSmtpHelo`.
Btw, shouldn't it be HELO instead of HELLO ? :)

Reply:
Sorry admittedly I'm a little dyslexic.
I pushed a commit that changes the flag to flagSmtpHelo, and updated its references.
I updated the readme and expected env var to SMTP_HELO.
Let me know if any other adjustments need to be made, please.